docs/SETUP_CI_CD_PIPELINE.md

```diff
@@ -489,6 +490,34 @@ directory so the playbooks can be performed from environment specific directory.
 This ansible script also updates the `gigadb_db_host` Gitlab variable with the 
 domain name of the RDS service in preparation for its provisioning.
 
+###### To enable provisioning dockerhost servers through a bastion server
+Bastion server provides perimeter access control, it acts as an entry point into a network containing private network instances. 
+Once dockerhost servers have been provisioned using terraform with ssh port restricted, which could then only be accessed the through bastion server. 
+
+Adding `ansible_ssh_common_args` in `/inventories/hosts` will make ansible to do the provisioning on dockerhost servers through bastion host.
+
+And prefixing the ansible commands with `TF_KEY_NAME=private_ip` to dockerhost_playbook.yml is essential as it would force dockerhost server to only accept a private ip entry,
+otherwise, `UNREACHEABLE !` would be occurred.
+
+##### How to manually ssh to dockerhost through the bastion for debugging purpose
```

Comment:
It's possible to SSH into dockerhost via the bastion server from your local dev machine this way:
```
$ ssh -i ~/.ssh/id-rsa.pem centos@10.**.*.*** -o ProxyCommand="ssh -W %h:%p -i ~/.ssh/id-rsa.pem centos@ec2-**-***-***-***.ap-northeast-1.compute.amazonaws.com"
Activate the web console with: systemctl enable --now cockpit.socket

Last login: Mon Nov 22 06:24:20 2021 from 10.99.0.24
[centos@ip-10-99-0-128 ~]$
```